ironfish-rust/src/keys/test.rs

```diff
@@ -112,3 +113,15 @@ fn test_hex_conversion() {
 
     assert!(PublicAddress::from_hex("invalid").is_err());
 }
+
+#[test]
+fn test_words_spending_key() {
+    let key = SaplingKey::generate_key();
+    let words = key
+        .words_spending_key(&bip39::Language::English)
+        .expect("Should return words");
+    let expected_words = MnemonicType::for_key_size(&key.spending_key[..].len() * 8)
```

Comment:
```suggestion
    let expected_words = MnemonicType::for_key_size(key.spending_key.len() * 8)
```

This just takes a `usize`, so taking a reference of a slice of an array is not needed